### DISCUSSION TITLE
Box Home Assistant Green and Skyconnect test - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#45] [Comments]
# Box Home Assistant Green and Skyconnect test - Haade.fr

Test of the Home assistant Green box at $99 and the Skyconnect zigbee dongle at $29.99, stability, performance there

https://haade.fr/en/blog/test-box-homeassistant-green-and-zigbee-key-skyconnect-performance-and-stability

**Comment by stefanuytterhoeven** [comment]
Skyconnect: it was sold as ZigBee dongle with the ability to add matter. But, now it looks like multiprotocol doesn't work reliability. So, if you want matter, add a new dongle 
Also, the box. For simple things, it's enough. But nabu casa is adding Ai voice.... Making ha bulky. The box will not be strong enough. It's good enough for starters

**Reply by haade-administrator** [reply]
I agree with you, after experience I am not for multi-protocol in a private home automation installation
